hphp/runtime/vm/jit/vasm-simplify.cpp

```diff
@@ -431,6 +431,21 @@ bool simplify(Env& env, const copyargs& inst, Vlabel b, size_t i) {
   });
 }
 
+bool simplify(Env& env, const loadb& inst, Vlabel b, size_t i) {
+  return if_inst<Vinstr::movzbl> (env, b, i+1, [&] (const movzbl& cm) {
+    // loadb; movzbl; -> loadzbl;
+    if (!(arch() == Arch::ARM &&
+          env.use_counts[inst.d] == 1 &&
+          env.use_counts[cm.s] == 1 &&
```

Comment:
Hi @mxw, I'm not sure which condition you are referencing.  The arch() line?  The whole 'if'?